common4j/src/main/com/microsoft/identity/common/java/commands/parameters/IBrokerTokenCommandParameters.java

```diff
@@ -97,6 +97,13 @@ public interface IBrokerTokenCommandParameters {
     @Nullable
     String getHomeTenantId();
 
+    /**
+     * Get the negotiated broker protocol version.
+     *
+     * @return a String representing broker protocol version
+     */
+    String getNegotiatedBrokerProtocolVersion();
```

Comment:
Where is this method going to be used?

Reply:
This was already there. We use `getNegotiatedBrokerProtocolVersion` in lots of places already to get the negotiated broker protocol version. All I did is added it directly to the interface. (The implementation is already present in child classes and being used).